api/src/API/Controllers/RaceResultsController.cs

```diff
@@ -90,26 +91,30 @@ public async Task<IActionResult> GetOneRaceResult(string orgId, string memberId,
         [HttpPost]
         public async Task<IActionResult> Create(string orgId, string memberId, RaceResult raceResult)
         {
-            if (raceResult.MemberId != Guid.Parse(memberId))
-            {
-                return BadRequest();
-            }
-
-            // Verify member and race exist
```

Comment:
Why are we not checking for this anymore?

Reply:
Good point, we should still check it